awx/main/dispatch/worker/callback.py

```diff
@@ -75,9 +75,8 @@ def read(self, queue):
         return {'event': 'FLUSH'}
 
     def record_read_metrics(self):
-        if self.queue_pop == 0:
```

Comment:
since we are piggy backing off the read method, we don't want to update uwsgi stats only when jobs are being processed, so we remove this condition.